lib/api/core/entrypoints/embedded/protocols/http.js

```diff
@@ -263,7 +263,10 @@ class HttpProtocol extends Protocol {
     debug('HTTP request response: %a', data);
 
     if (data.raw) {
-      if (typeof data.content === 'object') {
+      if (!data.content) {
```

Comment:
This test makes Kuzzle return a 0-length response if the content is falsey, such as  the number `0` or the `false` boolean value.

```suggestion
      if (data.content === null || data.content === undefined) {
```